src/main/java/com/redhat/rhjmc/containerjfr/MainModule.java

```diff
@@ -88,7 +89,7 @@ public static Gson provideGson() {
     @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
     @Provides
     @Named(RECORDINGS_PATH)
-    static Path provideSavedRecordingsPath() {
-        return Paths.get("/", "flightrecordings");
+    static Path provideSavedRecordingsPath(Logger logger, Environment env) {
```

Comment:
The `Logger` isn't used anymore. It can either be removed, or maybe there should be a message logged if the environment variable is (or isn't) set so that it's clear from the logs what path was used during a particular run.

Reply:
Ok, logger now reports set path.